Assets/Plugins/Notifications/Impls/Android/AndroidNotificationsService.cs

```diff
@@ -34,9 +35,13 @@ internal AndroidNotificationsService(Transform countlyGameObject, CountlyConfigu
             _bridge = gameObject.AddComponent<AndroidBridge>();
             _bridge.Log = Log;
 
-            AndroidJavaClass countlyPushPlugin = new AndroidJavaClass(CountlyPushPluginPackageName);
-            countlyPushPlugin.CallStatic("setEnableLog", config.EnableConsoleLogging);
-
+            try {
+                AndroidJavaClass countlyPushPlugin = new AndroidJavaClass(CountlyPushPluginPackageName);
+                countlyPushPlugin.CallStatic("setEnableLog", config.EnableConsoleLogging);
+            } catch (Exception ex) {
+                Log.Error("[AndroidNotificationsService] Exception: " + ex.Message);
+                throw new FileNotFoundException(ex.Message);

```

Comment:
This exception should never be thrown, why does this call exist here?

Reply:
on the base of this exception, ProxyNotificationServcie don't keep it's instance and  block all future request make to it.

                countlyPushPlugin.CallStatic("setEnableLog", config.EnableConsoleLogging);
                
                this code enable/disable the log in native android code.